src/cli/commands/utils/helpers.ts

```diff
@@ -55,3 +100,24 @@ ${exportInstruction} {
 ${separator}
 `;
 };
+
+export const runInstall = async (packageManager: 'yarn' | 'npm' | 'pnpm', pluginPath: string) => {
+  const execa = execa_({
+    cwd: pluginPath,
+    verbose: 'full',
+  });
+
+  await execa`${packageManager} install`;
+};
+
+export const runBuild = async (packageManager: 'yarn' | 'npm' | 'pnpm', pluginPath: string) => {
+  const execa = execa_({
+    pluginPath,
+    verbose: 'full',
+  });
+
+  if (packageManager === 'npm') {
+    await execa`${packageManager} run build`;
+  }
+  await execa`${packageManager} build`;
+};
```

Comment:
```suggestion

export const runInstall = async (packageManager: 'yarn' | 'npm' | 'pnpm', pluginPath: string) => {
  const { execa: execaPkg } = await import('execa');

  const execa = execaPkg({
    cwd: pluginPath,
    verbose: 'full',
  });

  await execa`${packageManager} install`;
};

export const runBuild = async (packageManager: 'yarn' | 'npm' | 'pnpm', pluginPath: string) => {
  const { execa: execaPkg } = await import('execa');

  const execa = execaPkg({
    pluginPath,
    verbose: 'full',
  });

  if (packageManager === 'npm') {
    await execa`${packageManager} run build`;
    return;
  }
  await execa`${packageManager} build`;
};

```

Needs to be an import because `execa` is esm

(and don't forget to remove the import from the top)